rbac/management/permission/view.py

```diff
@@ -130,7 +130,12 @@ def list(self, request, *args, **kwargs):
           ]
         }
         """
-        return super().list(request=request, args=args, kwargs=kwargs)
+        permission_list = super().list(request=request, args=args, kwargs=kwargs)
```

Comment:
The Django should have native support for the ordering. No need extra code. Could you check why the ordering_field is not working? https://github.com/RedHatInsights/insights-rbac/pull/1709/files#diff-c140a66d50961466350126cca95a5b9e7efa707093267242df6d21058e2595deL87-L88